src/Sylius/Bundle/ApiBundle/Resources/config/serialization/ShippingMethod.xml

```diff
@@ -41,6 +41,10 @@
             <group>shop:shipping_method:read</group>
         </attribute>
 
+        <attribute name="price">
+            <group>shop:shipping_method:read</group>
+        </attribute>
+
```

Comment:
Does it change anything?

Reply:
@lchrusciel this was left here by accident, I'll remove it. Price is added at normalization decorator